src/components/ui/social-login-button.tsx

```
@@ -0,0 +1,70 @@
+'use client'
+
+import { ButtonHTMLAttributes, forwardRef } from 'react'
+import { cn } from '@/lib/utils'
+
+export interface SocialLoginButtonProps extends ButtonHTMLAttributes<HTMLButtonElement> {
+  provider: 'google' | 'twitter' | 'github'
+}
+
+const providerIcons = {
+  google: (
+    <svg className="h-5 w-5" viewBox="0 0 24 24" aria-hidden="true">
```

ESLint

[error] Replace `"h-5·w-5"·viewBox="0·0·24·24"·aria-hidden="true"` with `'h-5·w-5'·viewBox='0·0·24·24'·aria-hidden='true'`

(prettier/prettier)